client/my-sites/domains/domain-management/settings/settings-header.tsx

```diff
@@ -0,0 +1,168 @@
+import { Circle, SVG } from '@wordpress/components';
+import { home, Icon, info } from '@wordpress/icons';
+import { useI18n } from '@wordpress/react-i18n';
+import { connect } from 'react-redux';
+import Badge from 'calypso/components/badge';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { isMappedDomain, resolveDomainStatus } from 'calypso/lib/domains';
+import { type as DomainType } from 'calypso/lib/domains/constants';
+import Breadcrumbs from 'calypso/my-sites/domains/domain-management/components/breadcrumbs';
+import {
+	domainManagementEdit,
+	domainManagementList,
+	isUnderDomainManagementAll,
+} from 'calypso/my-sites/domains/paths';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import type {
+	SettingsHeaderProps,
+	SettingsHeaderConnectedProps,
+	SettingsHeaderPassedProps,
+} from './types';
+import type { TranslateResult } from 'i18n-calypso';
+import './style.scss';
+
+const SettingsHeader = ( props: SettingsHeaderProps ) => {
+	const { selectedDomainName, isManagingAllDomains } = props;
+	const { __ } = useI18n();
+	let formattedHeaderText = selectedDomainName;
+	if ( ! selectedDomainName ) {
+		formattedHeaderText = isManagingAllDomains ? __( 'All Domains' ) : __( 'Site Domains' );
+	}
```

Comment:
Probably after removing Breadcrumbs from this component you could also remove this code, unwrap the component from Redux and remove `SettingsHeaderConnectedProps` 😉